### PR TITLE
Add different issue templates

### PR DESCRIPTION
**Pull request by LefterisJP**
[skip ci]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/critical_vulnerability_report.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: Critical Vulnerability Report
+about: Critical vulnerability reports about the Raiden Network
+---
+
+# DO NOT use Github
+
+If the issue you want to report is a critical vulnerability, something that may lead to lock up or loss of funds for Raiden users, then please do not use the public issue tracker.
+
+Instead send an email with a detailed report to bounty@raiden.network.
+TODO: Also add a link to the bug bounty website rules here.
```
Comment (CosminNechifor) [inline .github/ISSUE_TEMPLATE/critical_vulnerability_report.md]: Shouldn't we also tell the user to try reaching the Raiden team AFAP? Like personal chat on Gitter, or some other kind of communication platform?

Comment (ulope) [reply .github/ISSUE_TEMPLATE/critical_vulnerability_report.md]: It should be our responsibility to ensure that emails sent to this address are read quickly. The more communication channels we advertise the bigger the chance that something is going to get lost.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+---
+name: Bug Report
+about: Non critical bug reports about the Raiden Network
+---
+
+## Intro
```
Comment (ulope) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I'd suggest to put instructional parts of the issue text between html comments (i.e. `<!--` and `-->`) that way they won't show up in the rendered issue if the users forgets to remove them.

File: .github/ISSUE_TEMPLATE/critical_vulnerability_report.md (hunk 1 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/critical_vulnerability_report.md]: This email is the best way to reach us for security issues. It's sent to multiple people including myself. I scan it the minute I receive it.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/bug_report.md]: You mean the TODO?

Comment (ulope) [reply .github/ISSUE_TEMPLATE/bug_report.md]: No I meant the whole `Intro` section.

Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Oh I see. That's a nice idea.

Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Pushed the requested change. Will this work like you wanted it?

**Comment by hackaugusto** [comment]
no need to wait for travis on this

**Comment by LefterisJP** [comment]
@hackaugusto It was `[skip ci]`, which means it would wait forever :D

**Pull request merged**